src/operator/contrib/count_sketch.cu

```diff
@@ -0,0 +1,175 @@
+/*!
+ * Copyright (c) 2015 by Contributors
+ * \file count_sketch.cu
+ * \brief count_sketch op
+ * \author Chen Zhu, Yang Shi
+*/
+#include "./count_sketch-inl.h"
+#include <mshadow/tensor.h>
+#include <stdio.h>
+#include <algorithm>
+
+
+
+#define WARPS_PER_BLOCK 1
+#define THREADS_PER_BLOCK 512
+
+#define CUDA_KERNEL_LOOP(i, n) \
+  for (int i = blockIdx.x * blockDim.x + threadIdx.x; \
+       i < (n); \
+       i += blockDim.x * gridDim.x)
+namespace mshadow {
+namespace cuda {
+// wrappers to deal with atomic add
+// supporting only single precision
+__device__ void atomic_add(float* dst, float val) {
+atomicAdd(dst, val);
+}
+
+// for double precision
+__device__ void atomic_add(double* address, double val) {
+  // code example in the official document at:
+  // http://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html
+  //      #atomic-functions
+
+  // NOLINT_NEXT_LINE(runtime/int)
+  unsigned long long int* address_as_ull = (unsigned long long int*) address;  // NOLINT(*)
+  unsigned long long int old = *address_as_ull, assumed;                     // NOLINT(*)
+  do {
+    assumed = old;
+    old = atomicCAS(address_as_ull, assumed,
+            __double_as_longlong(val + __longlong_as_double(assumed)));
+    // Note: uses integer comparison to avoid hang in case of NaN
+    // (since NaN != NaN)
+  } while (assumed != old);
+}
+
+template <typename DType>
+__global__ void sketch_forward_kernel(const int nthreads, DType *out, const DType *h,
+                    const DType *s, const DType *in, const int n_smaples,
+                    const int in_dim, const int out_dim) {
+    // input: n_smaples * in_dim
+    // output: n_smaples * out_dim
+        const int index = blockIdx.x * blockDim.x + threadIdx.x;
```

Comment:
fix code style. Currently .cu files are not lint checked. But you should follow the same style guide